src/main/webapp/app/entities/course/course.service.ts

```diff
@@ -79,6 +84,11 @@ export class CourseService {
         return copy;
     }
 
+    private convertGenericFromServer(any) {
```

Comment:
Why do you need this method?

Reply:
Well, I saw that you retrieve the data from server and usually map them to models such as Course, but I don't think that I should do the same with the total score information I receive, right? So I thought this method can be used just to map my response to an entity unspecified.

Reply:
But now that I see I should probably map only the body of the response, I see that I give the complete response to it. Or you would suggest a better approach? :/